class.jetpack.php

```diff
@@ -4260,18 +4260,143 @@ function admin_page_load() {
 		add_filter( 'jetpack_short_module_description', 'wptexturize' );
 	}
 
-	function admin_notices() {
+	/**
+     * Converts an error_code into a user friendly string for admin notices
+     *
+	 * @param $error
+	 *
+	 * @return string
+	 */
+	private function pretty_error_codes( $error ) {
```

Comment:
Does moving these to PHP mean that we can also remove them from the .jsx file?

Reply:
I'm unsure on this part, haven't fully explored the react app to say whether there are any plan changes or other actions that would trigger errors to show in react.

Reply:
Without comment on how to do it (an endpoint, passing this through as vars and `wp_localize_script`, etc), I would like us to only have one source of truth for error->messaging. We've been bitten before by having two lists for the same thing in separate places get out of sync.